### PR TITLE
Spectrogram plot multiple plots

### PR DESCRIPTION
**Pull request by joaeba**
- add acceleration spectrogram plot 
- the spectrogram is calculated using fft and X, Y, Z components are accumulated

### CODE REVIEW

**Comment by joaeba** [comment]
ok, I've done the refactoring and I will still check the downsampling issue.

yes, it wrongly assumes regular sampling at the moment. I've tried resampling by linear interpolation before, but it actually created some artifacts and didn't improve anything on the approx. 5 logs which I visually inspected, hence I decided to go without any resampling. Probably, it would work, if I choose some more sophisticated resampling.

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Flight Review is deployed on https://review.px4.io.
 
 - clone the repository
 - use python3
-- `pip3 install bokeh jinja2 pyulog simplekml` (at least version 0.12.6
+- `pip3 install bokeh jinja2 pyulog simplekml pandas scipy` (at least version 0.12.6
```
Comment (bkueng) [inline README.md]: I think `pandas` can be removed from here and the travis.yml file (It was required at some point, but not anymore)

**Comment by joaeba** [comment]
I have now implemented downsampling to maximum 2*plot_width samples. With current plot_width at 800 that's 2 *800 * 4 bytes * 129 = 0.8 Mb. Do you think that's ok?

I still assume regular sampling, but it seems to work fine for a general overview. We can try resampling if we find a problem with that.

Thank you for all your help.

File: plot_app/plotting.py [hunk 2]
```diff
@@ -640,3 +649,91 @@ def _setup_plot(self):
         p.toolbar.logo = None
 
 
+class DataPlotSpec(DataPlot):
+    """
+    A spectrogram plot.
+    This does not do downsampling.
```
Comment (bkueng) [inline plot_app/plotting.py]: This comment needs to be updated: it does downsample, just not dynamic

Comment (joaeba) [reply plot_app/plotting.py]: ok, I've fixed the CI issue and updated the comment.

thank you again.

**Comment by bkueng** [comment]
Merged!

**Pull request closed without merging**